src/main/kotlin/step4/racingcar/racingcar/Car.kt

```diff
@@ -0,0 +1,25 @@
+package step4.racingcar.racingcar
+
+class Car(
+    val racerName: String
+) {
+    init {
+        if (racerName.length > RACER_NAME_LENGTH_LIMIT) {
+            throw IllegalArgumentException("입력된 선수의 이름이 5자 이상인 선수가 있습니다.")
+        }
```

Comment:
```suggestion
        require(racerName.length < RACER_NAME_LENGTH_LIMIT) { "입력된 선수의 이름이 5자 이상인 선수가 있습니다." }
```

kotlin에서는 require 함수를 통해서 IllegalArgumentException을 처리할 수 있습니다!

참고

[Kotlin 예외처리](https://seosh817.tistory.com/155)